03-Raster-Calculations-In-R.Rmd

```diff
@@ -82,12 +90,17 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
+
+###Sources of Additional Information
+<a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
+Read more about the `raster` package in R.</a>
+
 </div>
 
 #Raster Calculations in R
 We often want to perform calculations on two or more rasters to create a new 
 output raster. For example, if we are interested in mapping the heights of trees
-across an entire field site, we might want to calculate the *difference* between 
+across an entire field site, we might want to calculate the *difference* between
```

Comment:
does this bold? just noticed this. normally you need two ** 
*difference* **difference**